tdp/core/dag.py

```diff
@@ -23,6 +23,19 @@
 
 logger = logging.getLogger("tdp").getChild("dag")
 
+SERVICES_ORDER = {
+    "zookeeper": 1,
+    "hadoop": 2,
+    "ranger": 3,
+    "hdfs": 4,
+    "yarn": 5,
+    "hive": 6,
```

Comment:
As discussed, the explicitly numbered order of Hive, Hbase and Spark make sense that there are an dependency between them. It's better that they have the same number and are sorted by lexicographic order IMO.

Reply:
As discuss we will keep explicite priorities for each service with a different number.